src/themes/dark.tsx

```diff
@@ -4,6 +4,35 @@ const colors = {
 
 };
 
+const styles = {
+  "form": {
+    "button": {
+      "primary": {
+          "default": {
+              // "backgroundImage": "linear-gradient(135.00deg, blue 0%, green 100%)"
```

Comment:
I reallt think this is what the dark primary should be... :grinning:

Reply:
![image](https://user-images.githubusercontent.com/779444/124726833-14d45b00-df49-11eb-9cf2-00054df66e74.png)